go.mod

```
@@ -27,7 +27,7 @@ require (
 	github.com/bwmarrin/discordgo v0.23.0
 	github.com/bxcodec/faker/v3 v3.3.1
 	github.com/cenkalti/backoff v2.2.1+incompatible
-	github.com/coreos/go-oidc v2.2.1+incompatible
+	github.com/coreos/go-oidc/v3 v3.1.0
```

BTW, looks like there's a newer version now:
```suggestion
	github.com/coreos/go-oidc/v3 v3.5.0
```